src/ui/units/workbooks/components/Table/WorkbookEntriesTable/WorkbookEntriesTable.tsx

```diff
@@ -1,57 +1,56 @@
-import {DL} from 'constants/common';
-
 import React from 'react';
 
-import {dateTime} from '@gravity-ui/date-utils';
 import block from 'bem-cn-lite';
-import {EntryIcon} from 'components/EntryIcon/EntryIcon';
 import {I18n} from 'i18n';
-import {useInView} from 'react-intersection-observer';
 import {useDispatch} from 'react-redux';
-import {Link} from 'react-router-dom';
-import {WorkbookPage} from 'shared/constants/qa/workbooks';
+import {EntryScope} from 'shared';
 import {DIALOG_COPY_ENTRIES_TO_WORKBOOK} from 'ui/components/CopyEntriesToWorkbookDialog';
+import {CreateEntryActionType} from 'ui/units/workbooks/constants';
 
 import {GetEntryResponse} from '../../../../../../shared/schema';
 import {WorkbookWithPermissions} from '../../../../../../shared/schema/us/types';
-import {registry} from '../../../../../registry';
 import {AppDispatch} from '../../../../../store';
 import {closeDialog, openDialog} from '../../../../../store/actions/dialog';
 import {WorkbookEntry} from '../../../types';
 import {DIALOG_DELETE_ENTRY_IN_NEW_WORKBOOK} from '../../DeleteEntryDialog/DeleteEntryDialog';
 import {DIALOG_DUPLICATE_ENTRY_IN_WORKBOOK} from '../../DuplicateEntryDialog/DuplicateEntryDialog';
-import {EntryActions} from '../../EntryActions/EntryActions';
 import {DIALOG_RENAME_ENTRY_IN_NEW_WORKBOOK} from '../../RenameEntryDialog/RenameEntryDialog';
 
-import {ChunkItem, useChunkedEntries} from './useChunkedEntries';
+import {ChunkGroup} from './ChunkGroup/ChunkGroup';
+import {MainTabContent} from './MainTabContent/MainTabContent';
+import {defaultRowStyle} from './constants';
+import {useChunkedEntries} from './useChunkedEntries';
 
 import './WorkbookEntriesTable.scss';
 
 const i18n = I18n.keyset('new-workbooks');
 
 const b = block('dl-workbook-entries-table');
 
-const DATETIME_FORMAT = 'DD.MM.YYYY HH:mm:ss';
-const options = {
-    rootMargin: '200% 0px',
-};
-const ROW_HEIGHT = 48;
-const defaultRowStyle: React.CSSProperties = {
-    height: ROW_HEIGHT,
-};
-const onClickStopPropogation: React.MouseEventHandler = (e) => {
-    e.stopPropagation();
-    e.preventDefault();
-};
-
 type WorkbookEntriesTableProps = {
     workbook: WorkbookWithPermissions;
     entries: GetEntryResponse[];
-    refreshEntries: () => void;
+    refreshEntries: (scope: EntryScope) => void;
+    loadMoreEntries?: (entryScope: EntryScope) => void;
+    retryLoadEntries?: (entryScope: EntryScope) => void;
+    scope?: EntryScope;
+    mapTokens?: Record<string, string>;
+    mapErrors?: Record<string, boolean>;
+    mapLoaders?: Record<string, boolean>;
 };
 
 export const WorkbookEntriesTable = React.memo<WorkbookEntriesTableProps>(
-    ({workbook, entries, refreshEntries}) => {
+    ({
+        workbook,
+        entries,
+        refreshEntries,
+        retryLoadEntries,
+        loadMoreEntries,
+        scope,
+        mapTokens,
+        mapErrors,
+        mapLoaders,
+    }) => {
         const dispatch: AppDispatch = useDispatch();
 
```

Comment:
We have separate `getClaims` requests for every `userId`:

<img width="440" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/0e433cec-0cb5-48fc-beff-36db039ebde3">

For fix this issue, we need to add here:
```ts
import {getUserId} from 'shared/modules/user';
import {getResolveUsersByIdsAction} from 'ui/store/actions/usersByIds';

...

React.useEffect(() => {
    const resolveUsersByIds = getResolveUsersByIdsAction();
    const userIds = new Set<string>();
    entries.forEach((entry) => {
        userIds.add(getUserId(entry.createdBy));
    });

    dispatch(resolveUsersByIds(Array.from(userIds)));
}, [dispatch, entries]);
```

Reply:
add these logic to WorkbookEntriesTable.tsx